appendices/assignments.md

```diff
@@ -0,0 +1,197 @@
+GitHub Actions).
+
+The instructor will decide the exact details of what is required to complete this
+assignment and what topic or problems the package should focus on.
+For those who are doing this course on their own, the assignment is to create a
+package that should creates some custom summaries of the climate, either local,
+regional, national, or global. These summaries could include minimum and maximum
+historical temperatures, how many days of rainy or cloudy days, or rainfall.
+You can decide on which location to choose, but we recommend creating something
+for your own city or region. Climate data is available from at the
+[World Weather Information Service](https://worldweather.wmo.int/en/dataguide.html)
+website.
+
+For an example of what this might look like, see 
+TODO: Add the repo of the example climate package.
+
+## Second project: Each learner contributing to an existing package
+
+Individual-based, small improvements to existing package: Used as second 
+deliverable, due after "Community involvement". Should only build off of
+and improve an existing package, so doesn't need many milestones. Used to
+reinforce importance of having documentation that can involve the community.
+Could be the final exercises of chapters between Build Management and 
+Community Involvement.
+
+1. Choose a package to improve by either:
+    1. Selecting one from the list of packages provided by the instructor.
+    2. Selecting one based on a search of GitHub, based on the search
+    instructions provided by the instructor.
+2. Improve the package to better fit with best practices based on what you've learned.
+    - You don't need to update everything in the package, only what you can and
+    what you've learned so far.
+    - You can optionally submit a Pull Request of the changes, but this depends
+    on whether the maintainer is open to Pull Requests. If you want to submit a
+    Pull Request, check if an issue already exists for improving something
+    and comment in the Issue that you will try to address the issue in a Pull
+    Request. Otherwise, open a new Issue and ask if the maintainer would be open
+    to Pull Requests.
+
+## Third project: Each group creating a package
+
+This assignment is *optional* as it depends on multiple factors like class size,
+duration of course, or virtual vs in-person. This would be the third and final 
+deliverable for the learners. There are no final exercises in each chapter that
+learners need to complete related to this assignment. Exercises and the
+structure of the group work would be determined by the instructor.
+This assignment would best be due sometime after the teamwork lesson (REF).
+
+- Using the same process as was done with the first project assignment,
+complete a package as a team, using the tools taught in the team and community
+chapters (TODO: REF).
+
+## Rubric and expectations of the learner
+
+TODO: Can fill this out more as we work on book
```

Comment:
Yeah, sounds good. As an instructor, I'd love having a rubric that I could use/adapt.